openc3-cosmos-init/plugins/packages/openc3-cosmos-demo/vite-BigWidget.config.js

```diff
@@ -0,0 +1,26 @@
+import { defineConfig } from 'vite'
+import VitePluginStyleInject from 'vite-plugin-style-inject'
+import vue from '@vitejs/plugin-vue'
+
+const DEFAULT_EXTENSIONS = ['.mjs', '.js', '.ts', '.jsx', '.tsx', '.json']
+
+export default defineConfig({
+  build: {
+    outDir: 'tools/widgets/BigWidget',
+    emptyOutDir: true,
+    sourcemap: true,
+    lib: {
+      entry: './src/BigWidget.vue',
+      name: 'BigWidget',
+      fileName: (format, entryName) => `${entryName}.${format}.min.js`,
+      formats: ['umd'],
+    },
+    rollupOptions: {
+      external: ['vue', 'vuetify'],
+    },
+  },
+  plugins: [vue(), VitePluginStyleInject()],
+  resolve: {
+    extensions: [...DEFAULT_EXTENSIONS, '.vue'], // not recommended but saves us from having to change every SFC import
+  },
+})
```

Comment:
Do we need to generate this vite config file now?

Reply:
Yes.

Reply:
We should make a note to update all the templates to match the new Vue, Vuetify, and Vite syntax